stampzilla/installer/binary/releases.go

```diff
@@ -0,0 +1,53 @@
+package binary
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+)
+
+type Release struct {
```

Comment:
Can you put the releases stuff in own package? I think we might want to use that in multiple locations :)